assignments/assignment-8/index.html

```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>Timed Assignment</title>
+    </head>
+    <body>
+        <form>
+            <label for="fullNameInput">Your Full Name</label>
+            <input type="text" name="fullName" id="fullNameInput" />
+
+            <label for="EmailInput">Your Email</label>
+            <input type="email" name="email" id="emailInput" />
+
+            <br />
+
+            <label for="rangeInput">Rate your Confidence Level</label>
+            <input
+                type="range"
+                name="rating"
+                id="rangeInput"
+                min="1"
+                max="10"
+            />
+
+            <label for="ageInput">Your Age</label>
+            <input type="number" name="age" id="ageInput" min="18" max="99" />
+
+            <p>Select your Favourite Quiz Category</p>
+            <label for="checkboxInput">History</label>
+            <input type="checkbox" name="category" id="checkboxInput" />
+            <br />
+            <label for="checkboxInput">Science</label>
+            <input type="checkbox" name="category" id="checkboxInput" />
+            <br />
+            <label for="checkboxInput">Sport</label>
+            <input type="checkbox" name="category" id="checkboxInput" />
+            <br />
+            <label for="checkboxInput">Literature</label>
+            <input type="checkbox" name="category" id="checkboxInput" />
+
+            <h1>Quiz</h1>
+
+            <p>1.Lorem ipsum dolor sit amet consectetur adipisicing elit?</p>
+            <label for="trueSelect">True</label>
```

Comment:
You do not need to create a quiz. you need to create radio buttons to select preferred quiz format